.github/workflows/minikube-zap-test.yml

```diff
@@ -0,0 +1,44 @@
+# This is a basic workflow to help you get started with Actions
+
+name: DAST with ZAP on minikube
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  issues: write
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test-minikube:
+    name: DAST test with minikube and ZAP
+    runs-on: ubuntu-latest
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+      - uses: eLco/setup-vault@v1
+      - name: Start minikube
+        uses: medyagh/setup-minikube@master
+        with:
+          minikube-version: 1.29.0
+          driver: docker
+          kubernetes-version: v1.25.6
+      - name: Setup helm
+        uses: azure/setup-helm@v3.5
+        id: install
+      - name: Setup application
+        run: |
+          ./k8s-vault-minkube-start.sh
+      - name: ZAP Scan
+        uses: zaproxy/action-baseline@v0.7.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue_title: ZAP DAST scan findings
+          docker_name: "owasp/zap2docker-stable"
+          target: "http://localhost:8080"
```

Comment:
Is it possible to add/maintain a suppression file :-)?

Reply:
Yes, adding it